tools/ci_build/build.py

```diff
@@ -1097,6 +1097,15 @@ def adb_shell(*args, **kwargs):
 def run_training_python_frontend_tests(cwd):
     run_subprocess([sys.executable, 'onnxruntime_test_ort_trainer.py'], cwd=cwd)
     run_subprocess([sys.executable, 'onnxruntime_test_training_unit_tests.py'], cwd=cwd)
+    run_subprocess([
+        sys.executable, 'orttraining_test_transformers.py',
```

Comment:
Is it possible to name the files in a way we can identify whether they belong to frontend or backend? With the new frontnend coming, we will have to review all frontend tests andidentifying them is the first step :)

Reply:
All existing python tests are frontend tests. Almost all of them interactive with backend. So strictly speaking they are e2e test as well. Tests added here is no different from other python e2e tests. However, due to the fact that it takes long time and requires external data to run, we only run it with night Azure pipeline. Yes all these tests shall also run with the new frontend.